### DISCUSSION TITLE
Welcome to tei2html Discussions!

### DISCUSSION THREAD

**Discussion by jhellingman** [jhellingman/tei2html#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ildar** [comment]
Hmm, Github Discussions is a new kid in the hood. This is the first time I
encounter it.
Yet many github projects willing to "discuss" usually go to Gitter and
happy there. Why didn't you?
BTW, Gitter recently improved the Matrix integration. The Matrix itself
provides almost limitless access to and from other chat platforms incl.
Telegram, IRC, whatnot. IMHO definitely the platform of choice.

**Reply by jhellingman** [reply]
This is a new feature of GitHub, which they probably initiated when they saw the success of third parties jumping in this gap. For me, it is because I wasn't aware of gitter, and even now I know, it probably is most convenient to keep everything together in one account.

**Comment by ildar** [comment]
Sorry, I disagree.
Gitter is transparent to github accounts (and this is probably why many
github project go right there). So the good old Gitter is no farer then the
new "Discussion" feature.

**Comment by ildar** [comment]
Actually I'm not THAT interested to push someone somewhere. Just discussing
the discussion ))

**Comment by ildar** [comment]
And BTW I already see a bug: e-mail reply breaks the thread of discussion
putting reply to the top level. 😕

**Reply by jhellingman** [reply]
You're right!
